blockchain/chainio.go

```diff
@@ -1463,12 +1463,19 @@ func (b *BlockChain) createChainState() error {
 	// genesis block, use its timestamp for the median time.
 	numTxns := uint64(len(genesisBlock.MsgBlock().Transactions))
 	blockSize := uint64(genesisBlock.MsgBlock().SerializeSize())
+
+	// Calcultate the next stake difficulty.
+	nextStakeDiff, err := b.CalcNextRequiredStakeDifficulty()
```

Comment:
No need to calculate this when creating a chain state for the genesis block.  The next stake difficulty is already well known for the block after the genesis block.  In particular, it's `b.chainParams.MinimumStakeDiff`.